scipy/sparse/tests/test_base.py

```diff
@@ -59,6 +59,26 @@ def assert_array_equal_dtype(x, y, **kwargs):
     assert_(x.dtype == y.dtype)
     assert_array_equal(x, y, **kwargs)
 
+NON_ARRAY_BACKED_FORMATS = frozenset(['dok'])
```

Comment:
The style checker wants two empty lines between this and the code above.

Same goes for the `TEST_MATMUL` line below:

```
scipy/sparse/tests/test_base.py:62:1: E305 expected 2 blank lines after class or function definition, found 1
scipy/sparse/tests/test_base.py:84:1: E305 expected 2 blank lines after class or function definition, found 1
```